admin_manual/installation/example_centos.rst

```diff
@@ -82,13 +82,13 @@ Manually building redis/imagick (optional)
 
 ::
 
+    dnf config-manager --set-enabled PowerTools
```

Comment:
Probably also recommend installing `dnf-plugins-core` then upfront for `dnf config-manager`